src/backend/distributed/executor/local_executor.c

```diff
@@ -506,3 +505,72 @@ DisableLocalExecution(void)
 					  (superuser() ? PGC_SUSET : PGC_USERSET), PGC_S_SESSION,
 					  GUC_ACTION_LOCAL, true, 0, false);
 }
+
+
+void
+SetTaskQueryAndPlacementList(Task *task, Query *query, List *placementList)
```

Comment:
missing function comment

Reply:
The AndPlacementList part does not seem to contribute a whole lot since it's just an assignment and distracts from the main purpose of this function.